docs/installationguide.md

```diff
@@ -202,6 +202,7 @@ The ones relating specific JSON bindings are described in the following table.
 | IOTA_MQTT_AVOID_LEADING_SLASH | mqtt.avoidLeadingSlash  |
 | IOTA_MQTT_CLEAN               | mqtt.clean              |
 | IOTA_MQTT_CLIENT_ID           | mqtt.clientId           |
+| IOTA_MQTT_DISABLED            | mqtt.disabled           |
```

Comment:
config.js should be modified to include the new mqtt.disabled and amqp.disables settings (in commented way). Note config.js is used also somehow to document configurations.

Reply:
Fixed 04e88ac